website/content/docs/concepts/security/data-encryption.mdx

```diff
@@ -13,39 +13,42 @@ Systems](/docs/configuration/kms) that protect the base encryption keys used
 for various functions. This page describes the various KMS key purposes that
 Boundary supports and how they are used within the system.
 
-~> **Note:** Boundary does not currently support rotation of its internal keys, but this
-will be supported in a future version. External keys can be rotated so long as
-the original keys remain available for decryption; full support for rotating
-these will come in a future version as well.
-
 ## The `worker-auth-storage` KMS Key
 
 The `worker-auth-storage` KMS key is used by a [PKI
 Worker](/docs/configuration/worker/pki-worker) for storage of authentication
 keys. It is optional for PKI workers; if not specified the authentication keys
 will not be encrypted on disk. This is not used by KMS workers.
 
-## The `root` KMS Key and Per-Scope KEK/DEKs <sup>OSS Only</sup>
+## The `rootKey` KMS Key and Per-Scope KEK/DEKs <sup>OSS Only</sup>
 
 Following best practices of using different encryption keys for different
 purposes, Boundary has a number of encryption keys generated within each scope.
+Each key further contains versions, which hold the key material used to encrypt
```

Comment:
I'm slightly confused about this line. Is this saying that "versions" hold the key material? For some reason, when I first read it, it didn't make sense that "versions" itself holds key material, that it's just a version number.

Reply:
Yep, the versions of a key hold the key material, so that each "key" is more conceptual and the key version actually maps more 1:1 to an encryption key. Do you think I could rewrite this to clarify this better?